src/view/screens/Home.tsx

```diff
@@ -35,6 +41,32 @@ export function HomeScreen(props: Props) {
   const {data: preferences} = usePreferencesQuery()
   const {data: pinnedFeedInfos, isLoading: isPinnedFeedsLoading} =
     usePinnedFeedsInfos()
+  const currentStarterPack = useCurrentStarterPack()
+  const usedStarterPacks = useUsedStarterPacks()
+  const {setShowLoggedOut, requestSwitchToAccount} = useLoggedOutViewControls()
+
+  React.useEffect(() => {
+    if (currentStarterPack && !currentStarterPack?.initialFeed) {
+      // In test environments, the URL won't start with `https://bsky.app`, so we want to find it by the route
+      try {
+        const route = new URL(currentStarterPack.uri).pathname
+        const foundIndex = usedStarterPacks?.findIndex(p => p.includes(route))
+        if (foundIndex === -1) {
+          setShowLoggedOut(true)
+          requestSwitchToAccount({requestedAccount: 'starterpack'})
+        }
+      } catch {
+        // Don't need to handle anything here, just put the user on the home screen
```

Comment:
We should never have empty catches when there is _our_ code in the try — it'll swallow legit mistakes like reference/type errors.